README.md

```diff
@@ -82,7 +82,7 @@ What the attributes mean:
 - `patterns[].pattern`: Valid Go regular expression to match tags.
```

Comment:
This should be updated too.

Reply:
All done - merging here. Ping me if anything is still wrong.